hail_scripts/utils/clinvar.py

```diff
@@ -34,18 +35,19 @@ def download_and_import_latest_clinvar_vcf(genome_version: str) -> hl.MatrixTabl
         raise ValueError("Invalid genome_version: " + str(genome_version))
 
     clinvar_url = CLINVAR_FTP_PATH.format(genome_version=genome_version)
-    local_tmp_file_path = "/tmp/clinvar.vcf.gz"
```

Comment:
Ran into reliability issues with this code.  `local_tmp_file_path` was getting corrupted... also I didn't have `wget` and we don't need to shell exec.